biggerquery/cli.py

```diff
@@ -417,6 +460,19 @@ def _cli_deploy_image(args):
                         vault_secret=args.vault_secret
                         )
 
+def _cli_build_image(args):
+    cmd = 'python setup.py build_project --build-image' + (' --e' if args.export_image_to_file else '')
```

Comment:
`project_setup.py` instead of `setup.py`

Reply:
Done